packages/create-cloudflare/src/common.ts

```diff
@@ -536,3 +551,24 @@ export async function getProductionBranch(cwd: string) {
 
 	return "main";
 }
+
+/**
+ * Ensure that the commit message has newlines etc properly escaped.
+ */
+function prepareCommitMessage(commitMessage: string): string {
+	return JSON.stringify(commitMessage);
+}
+
+export function quoteShellArgs(args: string[]): string {
+	if (process.platform !== "win32") {
+		return quote(args);
+	} else {
+		// Simple quoting if there is a space (doesn't handle quotes and spaces together)
```

Comment:
nit: this comment needs updating since the code no longer only cares about spaces

Reply:
Fixed.